api/src/main/java/com/epam/pipeline/entity/quota/QuotaEntity.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2017-2021 EPAM Systems, Inc. (https://www.epam.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.pipeline.entity.quota;
+
+import com.epam.pipeline.dto.quota.QuotaGroup;
+import com.epam.pipeline.dto.quota.QuotaPeriod;
+import com.epam.pipeline.dto.quota.QuotaType;
+import com.epam.pipeline.entity.user.PipelineUser;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import java.util.List;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "quota", schema = "pipeline")
+@NoArgsConstructor
+@AllArgsConstructor
+public class QuotaEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Enumerated(EnumType.STRING)
+    private QuotaGroup quotaGroup;
+
+    @Enumerated(EnumType.STRING)
+    private QuotaType type;
+
+    @Enumerated(EnumType.STRING)
+    private QuotaPeriod period;
+
+    private String name;
+
+    private Long value;
+
+    @ManyToMany
+    @JoinTable(
+            name = "quotas_users",
+            schema = "pipeline",
+            inverseJoinColumns = { @JoinColumn(name = "user_id") },
+            joinColumns = { @JoinColumn(name = "quota_id") }
+    )
+    private List<PipelineUser> informedUsers;
```

Comment:
It should be an object similar to `NFSQuotaNotificationRecipient` or any SID entity (`name` and `isPrincipal` flag), as we need to support group/role notifications as well